ui/partials/main/thPinboardPanel.html

```diff
@@ -66,23 +66,30 @@
      title="{{!hasPinnedJobs() ? 'No jobs pinned' : ''}}">
   <div class="btn-group">
     <button class="btn btn-default btn-xs save-btn"
-          title="{{ !hasPinnedJobs() ? 'No job selected' : 'Save all classification data' }}"
-          ng-click="save()"
-          ng-disabled="!canSaveClassifications()">save
+            title="{{ !user.loggedin && !canSaveClassifications() ?
+                      'Not logged in / default classification data / no job pinned' : 'Save all classification data' }}"
```

Comment:
It's probably best to be completely explicit here about the reason the user can't perform the action, even at the cost of html readability:

```
title="{{ user.loggedin ? 'Not logged in' : !canSaveClassifications() : 'No data to save' : !hasPinnedJobs() : 'No job pinned' : 'Save all' }}"
```